### DISCUSSION TITLE
Bug - Message : "Empty response from server"

### DISCUSSION THREAD

**Discussion by baptistechretien** [ChatGPTNextWeb/NextChat#5531] [Q&A]
Hello,

When I use different versions of ChatGPT 4o or 4o mini, I get an error message that has been appearing since this morning:

{
"error": true,
"message": "empty response from server"
}

The problem only occurs when using ChatGPT 4o or 4o mini, but it works when using ChatGPT o1 Mini. For clarification, I am using the Openrouter.ai API, and there is no problem when using the official API.

I tried changing my internet connection and resetting Vercel, but nothing works.

Thank you in advance for your help.

Best regards,

**Comment by afredzhou** [comment]
Same issue
